bukkit/src/main/java/net/william278/huskhomes/network/RedisMessenger.java

```diff
@@ -27,7 +27,7 @@ public void initialize(@NotNull HuskHomes implementor) {
         redisWorker = new RedisWorker(implementor.getSettings());
         redisWorker.initialize();
 
-        CompletableFuture.runAsync(() -> {
+        new Thread(() -> {
```

Comment:
asyncronous call for .async().subscribe() is not necessary in this situation.
it is all managed by lettuce-core

Reply:
you should leave it alone without any CompletableFuture or thread